1306_expansion_statements/expansion-statements.md

```diff
@@ -469,6 +469,409 @@ void f() {
 
 ## Implementation experience
 
-TODO
+Bloomberg's Clang/P2996 fork (available [on Godbolt](https://godbolt.org/z/Yjv1dM4eK)) implements all features proposed by this paper. Expansion statements are enabled with the `-fexpansion-statements` flag (or with `-freflection-latest`).
 
 
+# Proposed wording
+
+## [basic.scope.pdecl] Point of declaration {-}
+
+Update paragraph 11 to specify the locus of an expansion statement:
```

Comment:
Fwiw, I don't think we need block headings like this in this paper. There isn't going to be that much wording. I think p2996 is just special in that regard. Looking at the rendered version, I think it's pretty distracting too.

Better to just here say:

> Update [basic.scope.pdecl]{.sref}, paragraph 11 to specify the locus of an expansion statement

etc.